analyzers/src/SonarAnalyzer.VisualBasic/Extensions/InterpolatedStringExpressionSyntaxExtensions.cs

```diff
@@ -27,5 +29,41 @@ public static class InterpolatedStringExpressionSyntaxExtensions
     {
         public static string GetContentsText(this InterpolatedStringExpressionSyntax interpolatedStringExpression) =>
             interpolatedStringExpression.Contents.JoinStr(null, content => content.ToString());
+
+        public static bool TryGetGetInterpolatedTextValue(this InterpolatedStringExpressionSyntax interpolatedStringExpression, SemanticModel semanticModel, out string interpolatedValue)
```

Comment:
Can this method be added in SonarAnalyzer.Common.Extensions?
Because it's identical to the CSharp one.

Reply:
@andrei-epure-sonarsource  any idea if there's a place to put this method to be reused for both VB and C#?

Reply:
we have `ILanguageFacade` and `SyntaxFacade` - can this help?

Reply:
I wrapped it there in a future PR to use it trough the Facade, but that is just to be able to invoke it from a language agnostic place. The implementation has to be language specific as the classes are different.

Reply:
Yes, we don't have a good workaround for that. I had a look at the extensions on `SonarAnalyzer.Common` and we don't have language/facade specific extensions there

Reply:
Ok, I got the time to spend on this. Indeed, we don't currently have extensions.

However, we generally avoid duplicating logic in our common rules. We move logic inside the Base class and use generics to overcome the problem.

I'll call Csaba to discuss